dissect/target/loaders/log.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+from typing import Callable
+
+from dissect.target import Target
+from dissect.target.filesystem import VirtualFilesystem
+from dissect.target.loader import Loader
+
+
+class LogLoader(Loader):
+    LOGS_DIRS = {
+        "EvtxPlugin": "sysvol/windows/system32/winevt/logs",
+        "EvtPlugin": "sysvol/windows/system32/config",
+    }
+
+    @staticmethod
+    def detect(path: Path) -> bool:
+        return False
+
+    def appendix(self, plugin_method: Callable) -> dict[str, str]:
+        return {"logs_dir": self.LOGS_DIRS[str(plugin_method.__self__.__class__.__name__)]}
```

Comment:
How does this work? Why not detect a directory with `.evt` or `.evtx`, map them to the default directories, and call it a day? Can you provide some comments or explanation on why this route was taken?

Reply:
That will not work since get_logs() will try to obtain paths from the registry in both evt.pyt and evtx.py. The only way to circumvent this is to pass a specific logs_dir, so that is what I am doing on behalf of the user. But maybe I am overlooking something, feel free to correct me if I am wrong.

Reply:
No, I see no reference to a default directory, instead it checks whether EVENTLOG_REGISTRY_KEY is set:

https://github.com/fox-it/dissect.target/blob/8c90fbd3b917ad9d36f01b98f6dc586f8e881da8/dissect/target/plugins/os/windows/log/evt.py#L54

Is this a bug?

Reply:
1 line up: https://github.com/fox-it/dissect.target/blob/8c90fbd3b917ad9d36f01b98f6dc586f8e881da8/dissect/target/plugins/os/windows/log/evt.py#L52

Reply:
Yes, but then at line 54 it queries the registry anyway, there is no condition that checks whether that last part can be skipped so is this correct behaviour?

Reply:
Ah, yes, that’s indeed a bug then. By all means go ahead and fix that!